get_deps.sh

```diff
@@ -5,21 +5,6 @@ HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 set -e
 [[ $VERBOSE == 1 ]] && set -x
 
-if [[ "$1" == "cpu" ]]; then
```

Comment:
Let's not remove this for the time being, until gen_deps.sh is stabilized (there's quite a few changes coming up with the ARM PR).

Reply:
Done